smart-contract/contracts/Calculations.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity ^0.6.6;
+// SPDX-License-Identifier: ISC
```

Comment:
Assumed licence based on package.json specified value. Not sure what the appropriate value should be